build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.kotlin_ktx_version = '1.2.0'
     ext.wordPressUtilsVersion = '1.40.0'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.53.0-alpha1'
+    ext.gutenbergMobileVersion = '3513-ba6208281a9129514b76b0093609e1b7b7c8ac16'
```

Comment:
question, this would be changed before merging, right?

Reply:
Yes we would update it to the correct version that contains the current PRs (3513) - info